.github/workflows/tiledb-cloud-py.yaml

```diff
@@ -151,11 +151,13 @@ jobs:
             --store-durations \
             --splitting-algorithm least_duration \
             --clean-durations \
+            -r xX \
           || .github/scripts/retry 2 \
             pytest -sv \
             --last-failed \
             --tb short \
-            --color yes
+            --color yes \
+            -r fxX
```

Comment:
To gather xpass/xfail info into a report section.

Reply:
I think we also want `E` in this collection, since the default value for `-r` is `fE`. Other than that this looks good!